daedalus_client/src/minecraft.rs

```diff
@@ -14,14 +14,15 @@ pub async fn retrieve_data(
     uploaded_files: &mut Vec<String>,
     semaphore: Arc<Semaphore>,
 ) -> Result<VersionManifest, Error> {
-    let old_manifest = daedalus::minecraft::fetch_version_manifest(Some(
+    let mut old_manifest = daedalus::minecraft::fetch_version_manifest(Some(
         &*format_url(&format!(
             "minecraft/v{}/manifest.json",
             daedalus::minecraft::CURRENT_FORMAT_VERSION
         )),
     ))
     .await
     .ok();
+    old_manifest = None;
```

Comment:
Clippy error here. is old manifest meant to be set to None after this? If so just a clippy error (value assigned to old manifest never used)